T: 5000
dynamic: False
dt: 0.01
# env: panda-reacher-acc-v0
robot_type: panda
n: 7
limits:
  low: 
    - -2.8973
    - -1.7628
    - -2.8973
    - -3.0718
    - -2.8973
    - -0.0175
    - -2.8973
  high:
    - 2.8973
    - 1.7628
    - 2.8973
    - 0.0698
    - 2.8973
    - 3.7525
    - 2.8973
r_body: 0.15
selfCollision:
  pairs:
    - [0, 6]
    - [0, 7]
    - [1, 6]
    - [1, 7]
    - [2, 6]
    - [2, 7]
# TODO topic
obstacles:
  obst0:
    geometry:
      position:
        - 0.7
        - 0.3
        - 0.6
      radius: 0.15
    type: sphereObstacle
    dim: 3
    low:
      position:
        - -1.0
        - -1.0
        - 0.0
      radius: 0.0
    high:
      position:
        - 1.0
        - 0.3
        - 1.0
      radius: 0.2

